app/src/lib/email/event-registration.ts

```diff
@@ -42,6 +43,18 @@ export const sendEventRegistrationUpdate = async (props: EventProps) => {
   return result;
 };
 
+export const sendEventCanceled = async (props: EventProps) => {
+  const icsFile = createIcsFile({ ...props, method: ICalCalendarMethod.CANCEL });
+  const mailParams = createMailParams({
+    ...props,
+    subject: `Kansellert: ${props.summary}`,
+    icsFile,
+  });
```

Comment:
Kansellering er bare at man får mail og at kalenderinvite blir oppdatert? Det skjer ingen faktiske endringer med arrangementet i seg selv? Er kanskje litt dumt at det ikke står noe sted i Sanity at arrangementet er kansellert 🤔 Da kan arrangør kanskje bli i tvil om det fungerte å kansellere

Reply:
Kanskje det f.eks. skulle blitt avpublisert automatisk? Og at tittelen endrer seg til (AVLYST) før tittel f.eks.? Og kanskje vi forresten skal kalle det for "Avlyst" i stedet for "Kansellert", er kanskje mer vanlig ord å bruke på norsk 😊

Reply:
Ja enig med begge delene! Implementerer avpublisering og endring i tittel om scope ikke blir for stort. Eventuelt gjør om innholdet til read only?

Endrer i hvert fall til avlyst! 😄

Reply:
Jaa supert 😄 Innholdet kan gjerne være read only også!

Reply:
read only er mulig MEN duplisering fører til at innholdet fortsatt er read only og det er litt kjipt

Reply:
Ja oki, er ikke så farlig med readonly egentlig